Alignment/MillePedeAlignmentAlgorithm/python/mpsvalidate/bigStructure.py

```diff
@@ -75,7 +75,8 @@ def plot(MillePedeUser, alignables, config):
                     if (abs(line.Par[big.data[i]]) != 999999):
                         # set name of the structure
                         big.histoAxis[i].SetBinLabel(
-                            big.binPosition[i], alignables.get_name_by_objid(line.ObjId))
+                            big.binPosition[i],
+                            str(line.Name) if len(line.Name) <= 13 else str(line.Name)[:12]+".")
```

Comment:
I just tuned it such that it fits the available space for the x-axis bin labels with the given text size